huey_monitor/tasks.py

```diff
@@ -34,3 +39,30 @@ def store_signals(signal, task, exc=None):
 
     task_model_instance.state_id = last_signal.pk
     task_model_instance.save(update_fields=('state_id',))
+
+
+@on_startup()
+def startup_handler():
+    """
+    "Change" task state to "unknown" on startup for every "executing" tasks.
```

Comment:
How does this work in a cluster? In our Boxine setup, handlers can start all the time during normal process, so why is it valid to modify _all_ existing tasks then?

Reply:
Yes, that's not "valid" for all tasks. This can be sown in the test project.

Because of this, i named is "work-a-round" ;)

Maybe we can try to check if "executing" tasks are alive?

Reply:
But with this "workaround", you have two problems, don't you? I.e. the hanging tasks and the now mislabeled ones. If we can somehow check whether a task is alive, that would be great!